geometry_msgs/Point centre
float64 strength
float64 active_radius